README.md

```diff
@@ -136,11 +163,30 @@ val df = spark.sqlContext.read.format("com.cognite.spark.datasource")
 df.createTempView("assets")
 
 // Create a new asset and write to CDP
-val someAsset = Seq(("99-BB-99999",99L,"This is another asset",Map("sourceSystem"->"MySparkJob"),99L))
-val someAssetDf = someAsset.toDF("name", "parentID", "description","metadata","id")
-someAssetDf
- .write
- .insertInto("assets")
+// Note that parentId, asset type IDs and asset type field IDs have to exist
+val someAsset = Seq(
+(
+99L, Seq(0L), 99L, "99-BB-99999", 2231996316030451L, "This is another asset",
+Seq((100L, // asset type ID
```

Comment:
While it's very useful to have an example with an asset type, I think we should keep the simple case. The example this replaces is very easy to understand, while this new one is quite intimidating and most people don't need to know about asset types.

Reply:
Well, now the one with an asset type is gone, I think we can have both! ;)
First the simple case, then a complex one with an asset type.